lib/features/wallet_screen/widget/waltet_screen_body.dart

```diff
@@ -0,0 +1,345 @@
+part of '../../feature.dart';
+
+class WalletScreenBody extends ConsumerWidget {
+  const WalletScreenBody({
+    super.key,
+  });
+
+ Widget _buildTransactionItem(
+  BuildContext context, {
+  required String description,
+  required String date,
+  required String amount,
+  Color? iconBgColor,
+  required bool isUpward,
+  bool showGreenAmount = false,
+}) {
+  return Container(
+    padding: const EdgeInsets.all(16),
+    decoration: BoxDecoration(
+      color: Colors.white,
+      borderRadius: BorderRadius.circular(16),
+    ),
+    child: Row(
+      children: [
+        Container(
+          padding: const EdgeInsets.all(8),
+          decoration: BoxDecoration(
+            color: iconBgColor,
+            borderRadius: BorderRadius.circular(12),
+          ),
+          child: Transform.rotate(
+  angle: isUpward ? -math.pi / -4 : math.pi / 4,
```

Comment:
what's the purpose of this when you have already imported the upward and downward arrows ?

Reply:
please use icons from the design and not flutter icons